sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_shared_key_policy_async.py

```diff
@@ -6,6 +6,7 @@
 import time
 import six
 from azure.core.pipeline.policies import SansIOHTTPPolicy
+from azure.core.pipeline import PipelineRequest
```

Comment:
since this import is only needed for typing, we should be able to do the following:
```suggestion
from __future__ import annotations
from typing import TYPE_CHECKING
if TYPE_CHECKING:
    from azure.core.pipeline import PipelineRequest
```